example-app/app/Http/Controllers/Api/CitiesByAirlineController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+
+class CitiesByAirlineController extends Controller
+{
+    public function index(Request $request)
+    {
+        $airlineId = $request->input('airline_id');
+        $cities = City::whereHas('airlines', function($query) use ($airlineId) {
+            $query->where('airline_id', $airlineId);
+        })->get();
```

Comment:
```suggestion
        $cities = City::whereHas('airlines', fn($query) => $query->where('airline_id', $airlineId))->get();
```

Just for you to know that in some cases you can use short syntax